Lib/Src/CDispatcher.cpp

```diff
@@ -205,8 +205,9 @@ void CDispatcher::processComChannels()
         while (mp_comchannels[channel]->isDataAvailable())
         {
             bool b_command_recognised;
-            etl::string<MAX_STRING_SIZE> command =
-                mp_comchannels[channel]->getData();
+            // TODO: Change this method to work with the future command parser
```

Comment:
I had to change this because getData no longer returns a string. I think this should be done by the parser.